backend/django/core/utils/util.py

```diff
@@ -231,59 +231,26 @@ def create_metadata_objects_from_csv(df, project):
             )
 
 
-def create_label_similarity_results(project):
-    """Insert data label similarity results into database using bulk_create."""
+def generate_label_embeddings(project):
+    """Create embeddings for each description of label."""
```

Comment:
One small comment, when rebuilding, there are a lot of lines like this in the docker logs:

Downloading: 100%|██████████| 232k/232k [00:00<00:00, 1.12MB/s]

Are these downloading pre-trained models? There are about 20 lines like this and they don't have labels as to what is being downloaded.

Reply:
Yes, that is the application downloading all of the necessary parts of the pertained model it's going to use.